web/styles/portico/activity.css

```diff
@@ -255,75 +271,38 @@ tr.admin td:first-child {
     padding-bottom: 15px;
 }
 
-.remote-form {
-    margin-bottom: 10px;
-}
-
 .support-realm-icon {
     max-width: 25px;
     position: relative;
+    vertical-align: middle;
     top: -2px;
 }
 
-.support-realm-status-form {
-    position: relative;
-    top: 20px;
-}
-
-.support-discount-form {
-    position: relative;
-    top: -50px;
-}
-
-.support-plan-type-form {
-    position: relative;
-    top: -10px;
-}
-
-.sponsorship-pending-form {
-    position: relative;
-    top: -25px;
-}
-
-.current-plan-details {
-    position: relative;
-    top: -45px;
-}
-
-.approve-sponsorship-form {
-    position: relative;
-    top: -40px;
-}
-
-.billing-modality-form {
-    position: relative;
-    top: -30px;
-}
-
-.downgrade-plan-form {
-    position: relative;
-    top: -75px;
-}
-
-.modify-plan-method-select {
-    width: auto;
+.scrub-realm-form,
+.realm-organization-type-form,
+.realm-subdomain-form,
+.downgrade-plan-form,
+.billing-modality-form,
+.approve-sponsorship-form,
+.current-plan-details,
+.sponsorship-pending-form,
+.support-plan-type-form,
+.support-discount-form,
+.support-realm-status-form,
+.remote-form,
+.current-plan-information {
+    margin-bottom: 10px;
 }
```

Comment:
This kinda suggests we might want a `support-form` class rather than a declaration giving 12 classes with the same styles; not a high priority if it's not a trivial tweak though, since it's an internal support page.